### ISSUE TITLE
Results on K49

### ISSUE THREAD

**Issue by janzd** [rois-codh/kmnist#8]
Hi, I've trained some models on Kuzushiji-49 dataset. I tested some heavier models, DenseNet and Shake-Shake to see if and how much they outperform the shallow ResNet architecture, and I also tested different batch sizes and different augmentation techniques (Mixup, Cutout). The Shake-Shake model is actually probably the same as hysts used to report his results.
I was able to reach a balanced accuracy of 98.29 on K49 with Shake-Shake, a large batch size, and cutout.
Here is my [repo](https://github.com/kurapan/pytorch_image_classification) with the results.

**Comment by hardmaru** [comment]
Thanks for running these experiments! Would be great additions to the table
in the repo

**Comment by janzd** [comment]
Should I send a PR with an updated table?

**Comment by mxbi** [comment]
Thanks a lot, and sorry about the delay!
I went ahead and added your best result using both Shake-Shake and Densenet to the table in the repo.
